uaclient/status.py

```diff
@@ -191,6 +193,13 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_INVALID_SERVICE_OP_FAILURE_TMPL = """\
 Cannot {operation} '{name}'
 For a list of services see: sudo ua status"""
+MESSAGE_UNEXPECTED_ERROR_DURING_OP_TMPL = (
+    """\
+Unexpected error(s) occurred during {operation}
+For more details, see the log: /var/log/ubuntu-advantage.log
+Please file a bug at: """
```

Comment:
Should we suggest using `ubuntu-bug` instead of pointing directly at the bug URL?